************************************************************************
* id: 766
* requirement: PY-29051: Fix breakpoints on lines that throw exceptions
*=======================================================================
* no changes, only the initial requirement, implements, and discussion.
*=======================================================================
************************************************************************
* type: implements
*-----------------------------------------------------------------------
* time: 2018-03-22T01:30:28Z
* content: PY-29051: Fix breakpoints on lines that throw exceptions
* codeUrl: github.com/JetBrains/intellij-community/pull/766/commits/0bcb1ad8f49c97ea8b81e9b0d30009f2a2708c76
************************************************************************
************************************************************************
* type: discussion
*-----------------------------------------------------------------------
* time: 2018-03-22T01:33:08Z
* content: @Elizaveta239 Please take a look. I think this is a pretty serious issue.
* time: 2018-04-24T22:34:06Z
* content: @Elizaveta239 Did you get a chance to look at this one?
* time: 2018-04-25T09:36:23Z
* content: Hi! Yes, it's already tested and reviewed and it will be included into one of the upcoming PyCharm 2018.2 EAPs.
************************************************************************
